apps/06Checkers.js

```diff
@@ -49,15 +55,59 @@ function Board() {
         console.log(string);
     }
 
-    // Your code here
+    this.checkers = [];
+
+    this.createCheckers = function() {
+        var white = [[0, 1], [0, 3], [0, 5], [0, 7],
+        [1, 0], [1, 2], [1, 4], [1, 6],
```

Comment:
It's good practice to keep anything on the right side of an assignment to the right of the equal sign, even if it must continue on a different line.